src/ui/Sort/Sort.ts

```diff
@@ -51,20 +49,28 @@ export class Sort extends Component {
      * - `@field ascending`/`@field descending`, where you must replace `field` with the name of a sortable field in your index (e.g., `data-sort-criteria="@size ascending"`).
      *
      * You can specify a comma separated list of sort criteria to toggle between when interacting with this component instance (e.g., `data-sort-criteria="date descending,date ascending"`).
-     * Interacting with this compnent instance will cycle through those criteria in the order they are listed in.
+     * You can specify multiple sort criteria to be used in the same request by separating them with a semicolon (e.g., `data-sort-criteria="@size ascending;date ascending"` ).
```

Comment:
We might want to leave an empty line between those lines, so it isn't just one paragraph